azure/README.md

```diff
@@ -36,22 +37,26 @@ For security reasons, initial configuration from another machine is disabled, bo
 
 1. SSH into your new VM:
     ```bash
-    ssh -i ~/.ssh/your-secret-key YOUR_USERNAME@captain.mycommunity.guardianconnector.net
+    ssh -i ~/.ssh/your-secret-key YOUR_USERNAME@mycommunity.guardianconnector.net
```

Comment:
Hm, it's not documented above to set up the A record for `mycommunity`. Only for `*.mycommunity`.  That's why this `captain.` prefix was there - though reallly it could be anything, for example `random123.mycommunity.guardianconnector.net`


So I don't think this change is in-line with the rest of the README. However I expect you probably do want `mycommunity.guardianconnector.net` to work (and we likeky have been setting it up all along), so maybe the change you really want is to include that up above.

OTOH this is a very "if you have your own domain name" question, and I don't know how applicable it would be.

My very mild preference is to keep `mycommunity.gc.net` out of this, and require you to specify a prefix, I guess?